DMX.Gatekeeper.Api/Models/Labs/LabDeviceStatus.cs

```diff
@@ -0,0 +1,12 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+namespace DMX.Gatekeeper.Api.Models.Labs
+{
+    public enum LabDeviceStatus
+    {
+        Online,
+        Offline
+    }
```

Comment:
I think the `LabDeviceStatus` and `LabStatus` properties should be swapped. The device itself is what can be reserved, not the lab itself. It might also be worth adding an `Unknown` status to both `LabDeviceStatus` and `LabStatus`.